soh/src/overlays/actors/ovl_En_Ge3/z_en_ge3.c

```diff
@@ -152,7 +152,8 @@ void EnGe3_WaitTillCardGiven(EnGe3* this, PlayState* play) {
 }
 
 void EnGe3_GiveCard(EnGe3* this, PlayState* play) {
-    if ((Message_GetState(&play->msgCtx) == TEXT_STATE_EVENT) && Message_ShouldAdvance(play)) {
+    if (GameInteractor_Should(VB_SKIP_GERUDO_MEMBERSHIP_TALK,
+                              (Message_GetState(&play->msgCtx) == TEXT_STATE_EVENT) && Message_ShouldAdvance(play))) {
```

Comment:
The VB name here should be renamed to something more descriptive of the vanilla behaviour inside. As far as I can see, it's basically ending the conversation, so maybe something like `VB_END_GERUDO_MEMBERSHIP_DIALOG` or something among those lines?